package.json

```diff
@@ -10,7 +10,7 @@
   "author": "Jacob Johnson",
   "license": "ISC",
   "dependencies": {
-    "express": "^4.14.1"
+    "express": "^4.15.3"
```

Comment:
You *really* should lock in your package versions with at least a '~' and a patch level version. I've experienced countless headaches from re-deploying an app, nothing else changed, but that damn `^` broke the app because apparently semver is dead, and people break APIs all the damn time. Easier to upgrade than to debug versioning!